dask/dataframe/core.py

```diff
@@ -2153,6 +2153,119 @@ def _skew_numeric(self, bias=True, nan_policy="propagate"):
             graph, name, num._meta_nonempty.skew(), divisions=[None, None]
         )
 
+    @derived_from(pd.DataFrame)
+    def kurtosis(
+        self, axis=None, fisher=True, bias=True, nan_policy="propagate", out=None
+    ):
+        """
+        .. note::
+
+           This implementation follows the dask.array.stats implementation
+           of kurtosis and calculates kurtosis without taking into account
+           a bias term for finite sample size, which corresponds to the
+           default settings of the scipy.stats kurtosis calculation. This differs
+           from pandas.
+
+           Further, this method currently does not support filtering out NaN
+           values, which is again a difference to Pandas.
+        """
+        axis = self._validate_axis(axis)
+        _raise_if_object_series(self, "kurtosis")
+        meta = self._meta_nonempty.kurtosis()
+        if axis == 1:
+            result = map_partitions(
+                M.kurtosis,
+                self,
+                meta=meta,
+                token=self._token_prefix + "kurtosis",
+                axis=axis,
+                enforce_metadata=False,
+            )
+            return handle_out(out, result)
+        else:
+            if self.ndim == 1:
+                result = self._kurtosis_1d(
+                    self, fisher=fisher, bias=bias, nan_policy=nan_policy
+                )
+                return handle_out(out, result)
+            else:
+                result = self._kurtosis_numeric(
+                    fisher=fisher, bias=bias, nan_policy=nan_policy
+                )
+
+            if isinstance(self, DataFrame):
+                result.divisions = (self.columns.min(), self.columns.max())
+
+            return handle_out(out, result)
+
+    def _kurtosis_1d(self, column, fisher=True, bias=True, nan_policy="propagate"):
+        """1D version of the kurtosis calculation.
+
+        Uses the array version from da.stats in case we are passing in a single series
+        """
+        # import depends on scipy, not installed by default
+        from ..array import stats as da_stats
+
+        if pd.Int64Dtype.is_dtype(column._meta_nonempty):
```

Comment:
Should we use a more general dtype check here than `Int64Dtype`? i.e. something like `pd.api.types.is_integer_dtype`?

If we have a nullable Pandas Series of `Int32`, `Int16`, or `Int8` (or any `UInt` type) this would return false and the `np.issubdtype` check would fail.

Reply:
I changed this. There are a few other method that use the same line as I did, for example [skew](https://github.com/dask/dask/blob/master/dask/dataframe/core.py#L2110) and [variance](https://github.com/dask/dask/blob/master/dask/dataframe/core.py#L2017). Might be beyond this PR's scope to adjust those too, but is probably something that should be done as well.